public/app/percona/pmm-dump/SendToSupportModal.tsx

```diff
@@ -27,19 +27,26 @@ export const SendToSupportModal: FC<ModalProps> = ({ onClose, dumpIds }) => {
     directory: '',
   };
 
-  const onSubmit = (values: SendToSupportForm) => {
-    dispatch(
-      sendToSupportAction({
-        sftp_parameters: {
-          user: values.user,
-          address: values.address,
-          password: values.password,
-          directory: values.directory || undefined,
-        },
-        dump_ids: dumpIds,
-      })
-    );
-  };
+  const onSubmit = useCallback(
+    (values: SendToSupportForm) => {
+      dispatch(
+        sendToSupportAction({
+          sftp_parameters: {
+            user: values.user,
+            address: values.address,
+            password: values.password,
+            directory: values.directory || undefined,
+          },
+          dump_ids: dumpIds,
+        })
+      );
+
+      if (!isLoading) {
```

Comment:
Is it possible ot hit the submit button when loading? If so, I think it would make sense to have submit disabled till the loading is completed.